.travis.yml

```diff
@@ -13,6 +13,7 @@ services:
 env:
   - TARGET=ansiblecheck
   - TARGET=internal-minimal
+  - TARGET=metapackages
```

Comment:
Do you think this is something we need to do on every build for now?

Reply:
Is there a good reason not to do it for every build?

Reply:
I don't mind either way since it's done in parallel and should not be too flaky. I was just wondering what was the motivation of running it by travis, since we do not consume build products from travis.

Reply:
Without this, it's basically dead code as there's no other mechanism to ensure it continues to work moving forward. Further, since we don't have a way to kick of specific tests based on the changes that are being made, it's either we test on all changes (even diffs that don't are irrelevant to the test being performed), or rely on reviews to enforce manual testing (which usually means tests don't get run).

IMO, more testing is better than less testing, so long as adding more tests doesn't negatively impact the test results (e.g. introduce flakey tests), or negatively impact the test latency (e.g. add long running tests for little benefit). In this case, I don't think this additional test does either, so it's a prime candidate for running on each PR. This is similar to how I chose to run the style checks (shfmt, shellcheck, ansiblelint) for each PR, even though those tests may be irrelevant for some changes.

Relying on peer reviews to enforce (and communicate) which tests need to be run for any given change is a poor system, imo, so I prefer to avoid that whenever possible; even if it means sometimes/often running irrelevant tests (that ultimately pass, of course).